src/plugins/home/public/application/components/sample_data_view_data_button.js

```diff
@@ -71,18 +71,6 @@ export class SampleDataViewDataButton extends React.Component {
     const dashboardPath = `/app/dashboards#/view/${this.props.overviewDashboard}`;
     const prefixedDashboardPath = this.addBasePath(dashboardPath);
 
-    if (this.props.appLinks.length === 0) {
-      return (
-        <EuiSmallButton
-          onClick={createAppNavigationHandler(dashboardPath)}
-          data-test-subj={`launchSampleDataSet${this.props.id}`}
-          aria-label={viewDataButtonAriaLabel}
-        >
-          {viewDataButtonLabel}
-        </EuiSmallButton>
-      );
-    }
-
```

Comment:
why did we remove a button?

Reply:
it's by mistake, I add it back in the new commit